src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -59,7 +59,7 @@ public void testCons() {
 
     @Test
     public void testDupString() {
-        Assert.assertEquals("ababab", Utils.dupString("ab", 3));
+        //removed test because method is being removed
         Assert.assertEquals("aaa", Utils.dupString('a', 3));
```

Comment:
delete this line testing `dupString` and the comment, let all vestiges of `dupString` be obliterated!